Userland/Libraries/LibGUI/Window.cpp

```diff
@@ -522,6 +533,25 @@ void Window::propagate_shortcuts(KeyEvent& event, Widget* widget, ShortcutPropag
     event.ignore();
 }
 
+void Window::remember_state(DeprecatedString const& domain, DeprecatedString const& group)
+{
+    m_remember_state = true;
+    m_remember_state_domain = domain;
+    m_remember_state_group = group;
+
+    // Try to read window state from config
+    int x = Config::read_i32(domain.view(), group.view(), "X"sv, INT_MIN);
+    int y = Config::read_i32(domain.view(), group.view(), "Y"sv, INT_MIN);
+    if (x != INT_MIN && y != INT_MIN)
+        move_to(x, y);
+    int width = Config::read_i32(domain.view(), group.view(), "Width"sv, INT_MIN);
+    int height = Config::read_i32(domain.view(), group.view(), "Height"sv, INT_MIN);
+    if (width != INT_MIN && height != INT_MIN)
+        resize(width, height);
+    set_maximized(Config::read_bool(domain.view(), group.view(), "Maximized"sv, false));
+    set_fullscreen(Config::read_bool(domain.view(), group.view(), "Fullscreen"sv, false));
```

Comment:
I'm not sure we should be saving/restoring full screen state.

Reply:
Sure